features/networking/multicast.feature

```diff
@@ -273,3 +275,344 @@ Feature: testing multicast scenarios
+      | resource     | netnamespace    |
+      | resourcename | <%= cb.proj1 %> |
+      | overwrite    | true            |
+      | keyval       | netnamespace.network.openshift.io/multicast-enabled=true |
+    Then the step should succeed
+
+    # enable multicast and create testing pods in default project
+    Given I switch to cluster admin pseudo user
+    And I use the "default" project
+    Given admin adds and overwrites following annotations to the "default" netnamespace:
+      | netnamespace.network.openshift.io/multicast-enabled=true |
+    Then the step should succeed
+
+    Given admin ensures "mcast-rc" rc is deleted after scenario
+    When I run oc create over "<%= BushSlicer::HOME %>/testdata/networking/multicast-rc.json" replacing paths:
+      | ["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    Given 1 pods become ready with labels:
+      | name=mcast-pods |
+    And evaluation of `pod.ip` is stored in the :proj2_podip clipboard
+    And evaluation of `pod.name` is stored in the :proj2_pod clipboard
+
+    # run omping on the pod in first project
+    Given I use the "<%= cb.proj1 %>" project
+    When I run the :exec background client command with:
+      | pod              | <%= cb.proj1_pod %> |
+      | oc_opts_end      |                     |
+      | exec_command     | sh                  |
+      | exec_command_arg | -c                  |
+      | exec_command_arg | omping -c 5 -T 15 <%= cb.proj1_podip %> <%= cb.proj2_podip %> > /tmp/p1.log |
+    Then the step should succeed
+
+    # run omping on the pod in default project
+    Given I use the "default" project
+    When I run the :exec background client command with:
+      | pod              | <%= cb.proj2_pod %> |
+      | oc_opts_end      |                     |
+      | exec_command     | sh                  |
+      | exec_command_arg | -c                  |
+      | exec_command_arg | omping -c 5 -T 10 <%= cb.proj1_podip %> <%= cb.proj2_podip %> > /tmp/p2.log |
+    Then the step should succeed
+
+    # check the result and should receive 0 multicast packet
+    Given 10 seconds have passed
+    Given I use the "<%= cb.proj1 %>" project
+    When I execute on the "<%= cb.proj1_pod %>" pod:
+      | cat | /tmp/p1.log |
+    Then the step should succeed
+    And the output should contain:
+      | joined (S,G) = (*, 232.43.211.234), pinging |
+      | multicast, xmt/rcv/%loss = 5/0/100%         |
+
+    Given I use the "default" project
+    When I execute on the "<%= cb.proj2_pod %>" pod:
+      | cat | /tmp/p2.log |
+    Then the step should succeed
+    And the output should contain:
+      | joined (S,G) = (*, 232.43.211.234), pinging |
+      | multicast, xmt/rcv/%loss = 5/0/100%         |
+      
+  # @author hongli@redhat.com
+  # @case_id OCP-12928
+  @admin
+  Scenario: pods should be able to join multiple multicast groups at same time
+    Given the env is using multitenant or networkpolicy network
```

Comment:
@weliang1 I would actually rewrite this step and absorb the `or` logic within the step instead of having the user write `multitenant or networkpolicy`

Reply:
@pruan-rht The step "Given the env is using multitenant or networkpolicy network" is an existing step in each old test cases in both cucushift and verification-tests, no new changes did in this PR for this step.
https://github.com/openshift/cucushift/blob/master/networking/multicast.feature#L7
https://github.com/openshift/verification-tests/blob/master/features/networking/multicast.feature#L7